CPAC/utils/configuration/yaml_template.py

```diff
@@ -118,10 +118,10 @@ def dump(self, new_dict, parents=None):
             if 'pipeline_setup' not in new_dict:
                 new_dict['pipeline_setup'] = None
             # Insert automatically-changed value in original dict
-            if freesurfer_extraction:
-                new_dict = set_nested_value(
-                    new_dict, ['surface_analysis', 'freesurfer',
-                    'run_reconall'], False)
+            # if freesurfer_extraction:
+            #     new_dict = set_nested_value(
+            #         new_dict, ['surface_analysis', 'freesurfer',
+            #         'run_reconall'], False)
```

Comment:
I think we can just delete all of this (instead of commenting out these 4 lines):

- [`CPAC/surface/globals.py`](https://github.com/FCP-INDI/C-PAC/blob/4f4d6b17c54a1f1eebcf3dc06f8565338436dac5/CPAC/surface/globals.py) the whole file
- https://github.com/FCP-INDI/C-PAC/blob/4f4d6b17c54a1f1eebcf3dc06f8565338436dac5/CPAC/utils/configuration/configuration.py#L25
- https://github.com/FCP-INDI/C-PAC/blob/4f4d6b17c54a1f1eebcf3dc06f8565338436dac5/CPAC/utils/configuration/yaml_template.py#L23 https://github.com/FCP-INDI/C-PAC/blob/4f4d6b17c54a1f1eebcf3dc06f8565338436dac5/CPAC/utils/configuration/yaml_template.py#L120-L130